tutorials/chapter1.md

```diff
@@ -350,6 +350,61 @@ With this code, we have ensured that any vector published into the eventloop wil
 
 The last argument, a clone function that the publisher will use instead of standard copy, is also to increase performance. This is covered in depth in the [vision tutorial](https://github.com/klepsydra-technologies/kpsr-vision-ocv-tutorial).
 
+### Benchmark
+
+The next benchmark measures and shows the eventloop performance:
```

Comment:
We have provided a small benchmarking example to demonstrate the performance benefits of eventloop. The main benchmarking block is shown below, and the complete benchmark is in file `example4c.cpp`

Reply:
Done